demos/k8s-demo/chart/demo-app-secretless/README.md

```diff
@@ -0,0 +1,152 @@
+# demo-app-secretless
+
+[The Secretless Broker](https://github.com/cyberark/secretless-broker) is a connection broker which relieves client applications of the need to directly handle secrets to target services such as databases, web services, SSH connections, or any other TCP-based service.
+
+## TL;DR;
+
+```bash
+$ helm install -f values.yaml .
+```
+
+## Introduction
+
+This chart bootstraps a deployment of a configurable demo-application, which uses [PostgreSQL](https://github.com/docker-library/postgres) as its backend, with Secretless on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager. Secretless seamlessly manages the database connection for this application.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release \
+ --set applicationDBAddress="${DB_URL}" \
+ --set applicationDBUsername="${DB_USER}" \
+ --set applicationDBPassword="${DB_INITIAL_PASSWORD}"
+ .
+```
+
+The command deploys PostgreSQL on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the PostgreSQL chart and their default values.
+
+| Parameter                     | Description                                     | Default                                                    |
+| -----------------------       | ---------------------------------------------   | ---------------------------------------------------------- |
+| `applicationDBAddress`        | `postgres` database URL, e.g. db.svc.cluster.local:5432/table-name | `nil` (required)                                           |
+| `applicationDBUsername`       | Application database username                   | `nil` (required)                                           |
+| `applicationDBPassword`       | Application database password                   | `nil` (required)                                           |
+| `applicationPort`             | Image pull secrets                              | `nil` (required)                                           |
```

Comment:
Why isn't this included in the command above if it's "required"?

What does the "Image pull secrets" description mean? This doesn't seem valid for `applicationPort`